### PR TITLE
Add Training Wheel 40

### PR DESCRIPTION
**Pull request by KnoblesseOblige**
## Description

Added Training Wheel 40 keyboard by P3D

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by KnoblesseOblige** [comment]
- Fixed `info.json` not including all possible layouts
- Fixed `tw40.h` not including all possible layouts

File: keyboards/tw40/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# tw40
+
+![tw40](imgur.com image replace me!)
```
Comment (keyboard-magpie) [inline keyboards/tw40/readme.md]: Does this need tidying up?

Comment (KnoblesseOblige) [reply keyboards/tw40/readme.md]: I've resolved that in commit a8ea863

**Review by ridingqwerty — APPROVED** [review]
Consider adding reset instructions to the readme, but looks good to me otherwise.

File: keyboards/tw40/readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Training Wheel 40
+
+![tw40](https://cdn.shopify.com/s/files/1/0501/7537/7592/products/tw40_600x.jpg?v=1607835284)
+
+* Keyboard Maintainer: [KnoblesseOblige](https://github.com/yourusername)
+* Hardware Supported: Training Wheel 40 PCB
+* Hardware Availability: https://p3dstore.com/products/tw40-gb
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/tw40/readme.md:9]: Could you add instructions on how to enter the bootloader to flash new firmware?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/tw40/readme.md (hunk 2 above)
Comment (KnoblesseOblige) [reply keyboards/tw40/readme.md:9]: I've updated the README to include reset instructions. I couldn't find any prior art so hopefully my brief sentence is sufficient.

**Review by drashna — APPROVED** [review]

**Pull request merged**